File: src/main/intellibitz/sted/actions/OpenSampleFontMapAction.java
package sted.actions;

import sted.STEDGUI;
import sted.ui.STEDWindow;
import sted.ui.TabDesktop;

import javax.swing.*;
import java.awt.event.ActionEvent;

public class OpenSampleFontMapAction
        extends STEDWindowAction {
    public OpenSampleFontMapAction() {
        super();
    }

    public void actionPerformed(ActionEvent e) {
        final STEDWindow stedWindow = getSTEDWindow();
        STEDGUI.busy();
        final TabDesktop tabDesktop =
                stedWindow.getDesktop();
//        if (JOptionPane.CANCEL_OPTION != tabDesktop.saveDirty())
//        {
        final String fileName = (String) getValue(Action.NAME);
        tabDesktop.reopenFontMap(fileName);
//        }
        STEDGUI.relax();
    }

}

File: src/main/intellibitz/sted/actions/OpenSampleFontMapAction.kt
package sted.actions

import sted.STEDGUI
import java.awt.event.ActionEvent

class OpenSampleFontMapAction : STEDWindowAction() {
    override fun actionPerformed(e: ActionEvent) {
        STEDGUI.busy()
        val tabDesktop = stedWindow.desktop
        //        if (JOptionPane.CANCEL_OPTION != tabDesktop.saveDirty())
//        {
        val fileName = getValue(NAME) as String
        tabDesktop.reopenFontMap(fileName)
        //        }
        STEDGUI.relax()
    }
}